guides/doc-Release_Notes/topics/foreman-contributors.adoc

```diff
@@ -0,0 +1,37 @@
+We'd like to thank the following people who contributed to the Foreman {ProjectVersion} release:
+
+Adam Lazik,
+Adam Růžička,
+Archana Kumari,
+Bernhard Suttner,
+Cole Higgins,
+Eric Helms,
+Evgeni Golov,
+Ewoud Kohl van Wijngaarden,
+Francesco Di Nucci,
+Girija Soni,
+Gordon Bleux,
+Jan Bundesmann,
+Jason,
+Jeremy Lenz,
+Leos Stejskal,
+Manisha Singhal,
+Marcel Kühlhorn,
+Marek Hulán,
+Maria Agaphontzev,
+Markus Bucher,
+Markus Reisner,
+Matthias Dellweg,
+Nadja Heitmann,
+Nofar Alfassi,
+Oleh Fedorenko,
+Pablo Fernández Rodríguez,
+Patrick Creech,
+Pavel Moravec,
+Thorben Denzer,
+Tim Meusel,
+Waldirio M Pinheiro,
+akumari,
+kmalyjur
```

Comment:
In previously releases we have listed @kmalyjur with her full name. For now I think it's fine since at least there's no duplicate so that's fine.

@kmalyjur would you like to be credited with your full name or just the username?